aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -6,6 +6,8 @@
 #include <ATen/native/ReduceOpsUtils.h>
 #include <c10/util/Exception.h>
 #include <ATen/native/cpu/TensorCompareKernel.h>
+#include <ATen/native/cpu/Loops.h>
+#include <ATen/cpu/vec256/vec256.h>
```

Comment:
is this import necessary?

Reply:
I removed this, and change implementation to parallel